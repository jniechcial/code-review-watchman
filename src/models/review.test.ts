import Review from './review';

const stubbedData = {
    user: { login: 'test1' },
    state: 'CHANGES_REQUESTED',
};

describe('Review', () => {
    it('creates review properly', async () => {
        const pullRequest = new Review(stubbedData);
        expect(pullRequest.username).toEqual('test1');
    });
});
