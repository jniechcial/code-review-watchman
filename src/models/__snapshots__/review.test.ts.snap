// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review creates review properly 1`] = `
PullRequestReview {
  "state": "CHANGES_REQUESTED",
  "username": "test1",
}
`;
